restdocs-api-spec-jsonschema/src/main/kotlin/com/epages/restdocs/apispec/jsonschema/JsonSchemaFromFieldDescriptorsGenerator.kt

```diff
@@ -167,37 +168,11 @@ class JsonSchemaFromFieldDescriptorsGenerator {
     }
 
     private fun handleEndOfPath(builder: ObjectSchema.Builder, propertyName: String, fieldDescriptor: FieldDescriptorWithSchemaType) {
-
-        if (fieldDescriptor.ignored) {
-            // We don't need to render anything
-        } else {
+        if (!fieldDescriptor.ignored) {
             if (isRequired(fieldDescriptor)) {
                 builder.addRequiredProperty(propertyName)
             }
-            if (propertyName == "[]") {
```

Comment:
How is this case obsolete now?

Reply:
Okay, let's keep it removed. If it creates a regression, we get to know what wasn't tested. :sweat_smile: